api/client/help.go

```diff
@@ -7,6 +7,11 @@ import (
 	flag "github.com/docker/docker/pkg/mflag"
 )
 
+// CmdHelp displays information on a Docker command.
+//
+//If more than one command is specified, information is only shown for the first command.
```

Comment:
There are a few missing whitespace at the beginning of lines like that one, but not a blocker for me.